app/helpers/content_helper.rb

```diff
@@ -3,7 +3,14 @@ module ContentHelper
   # @param markdown [String]
   # @return [String]
   def translate_markdown(markdown)
-    raw Govspeak::Document.to_html(markdown, sanitize: false)
+    output = Govspeak::Document.to_html(markdown, sanitize: false)
+
+    if output.exclude?("class='govuk-link'")
```

Comment:
Instead, can we try applying `govuk-link` styling to any `a` tag within the `.gem-c-govspeak` class,  so this all happens in the CSS?

Reply:
I've updated it now